ui-admin/src/portal/siteContent/SiteContentView.tsx

```diff
@@ -1,9 +1,9 @@
 import React, { useState } from 'react'
-import { HtmlPage, NavbarItem, PortalEnvironment } from '../../api/api'
+import { HtmlPage, NavbarItemInternal, PortalEnvironment } from '../../api/api'
```

Comment:
Not your change, but we should use abosulte imports except where the files are closely related
```suggestion
import { HtmlPage, NavbarItemInternal, PortalEnvironment } from 'api/api'
```

Reply:
I think existing code is wildly inconsistent about this. Looks like a case for [eslint-plugin-import](https://github.com/import-js/eslint-plugin-import/blob/HEAD/docs/rules/no-relative-parent-imports.md).